ui/lib/pki/addon/components/pki-generate-root.ts

```diff
@@ -46,6 +48,7 @@ interface Args {
  * @callback onSave - Optional - Callback triggered after model is saved, as a side effect. Results are shown on the same component
  * @callback onComplete - Callback triggered when "Done" button clicked, on results view
  * @param {Object} adapterOptions - object passed as adapterOptions on the model.save method
+ * @param {Boolean} hideAlertBanner - hide alert banner that shows warning about private key material (used when form is rendered inside a parent that displays an alert banner)
```

Comment:
interesting.. this feels a little 🍝 , should we just consistently show the alert banner from the parent rather than doing it this way?

Reply:
I was going to ask you about that! I think that makes more sense - just didn't want to refactor that component before chatting with you!